_pages/OutsideRange.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: OutsizeRange MACRO
+permalink: /outside-range/
+exclude: true
+---
+
+![OutsideRange]({{'/puml/svgs/DetailedDesign-OutsideRange.svg' | relative_url}})
```

Comment:
Please wrap the image in a link to itself, this way we can click on it and have it full - size, or browser filling for that matter.

Reply:
done